build/BuildToolsTasks/BuildMGCBEditorTask.cs

```diff
@@ -25,6 +25,13 @@ public override void Run(BuildContext context)
             _ => "Linux"
         };
 
+        var runtime = context.Environment.Platform.Family switch
+        {
+            PlatformFamily.Windows => context.DotNetPublishSettings.Runtime = "win-x64",
```

Comment:
You are modifying the global context settings, this will cause issues for any step that happens after mgcb editor to use these properties.

Also checking the family will cause issues for building the linux mgcb editor on macOS / Windows.

Instead modify the packaging for each specific mgcb-editor variant to have specified runtime if it fixes the sizing issue.